testdata/p4_16_samples_outputs/bitvec_hdr-first.p4

```diff
@@ -0,0 +1,106 @@
+#include <core.p4>
```

Comment:
if you delete bitvec_hdr please delete all the associated reference outputs.

Reply:
Deleted bitvec_hr and all the associated reference outputs.  Thanks for the clarification on p4test and bmv2 file.

Reply:
This file is still part of the PR. I don't think that the conversation can be marked as resolved.
Please run git diff and check that everything that should not be part of the PR has been removed.

Reply:
Sorry, I forgot to delete the three files in the p4_16_samples_outputs dir.   

I just checked in a deletion of the three files.  

I will also double-check any other files that need deletion or any update.  thanks.